server/handlers.go

```diff
@@ -465,6 +465,8 @@ func (s *Server) sendCodeResponse(w http.ResponseWriter, r *http.Request, authRe
 		// Only valid for implicit and hybrid flows.
 		idToken       string
 		idTokenExpiry time.Time
+
+		accessToken = storage.NewID()
```

Comment:
This accessToken should be reused at line 516:

    v.Set("access_token", accessToken)

instead of

    v.Set("access_token", storage.NewID())